### PR TITLE
Add FIREWATCH success configuration

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

File: ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__opentelemetry-product-ocp-4.15-lp-interop.yaml [hunk 1]
```diff
@@ -63,6 +63,10 @@ tests:
               {"step": "gather-*", "failure_type": "all", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
               {"step": "distributed-tracing-tests-*", "failure_type": "pod_failure", "classification": "OpenTelemetry e2e tests", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
               {"step": "distributed-tracing-tests-*", "failure_type": "test_failure", "classification": "OpenTelemetry e2e tests", "jira_project": "TRACING", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]}
+            ],
+          "success_rules":
```
Comment (IshwarKanse) [inline ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__opentelemetry-product-ocp-4.15-lp-interop.yaml:67]: What does the success_rules do here.

Comment (Sau1506mya) [reply ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__opentelemetry-product-ocp-4.15-lp-interop.yaml:67]: @IshwarKanse , it is basically for reporting a success in a successful run.
Detailed information: https://github.com/RedHatQE/firewatch/blob/main/docs/configuration_guide.md#success_rules-optional

**Review by IshwarKanse — APPROVED** [review]

**Pull request merged**